gcloud/dns/changes.py

```diff
@@ -221,7 +221,8 @@ def create(self, client=None):
         self._set_properties(api_response)
 
     def exists(self, client=None):
-        """API call:  test for the existence of the change set via a GET request
+        """API call:  test for the existence of the change set via a GET
+        request.
```

Comment:
(This particular case might be able to be reduced to one string, so just consider this a meta conversation)

Considering that the community has gone one way, I'd argue that PEP257 is what's broken. Practicality over dogma. The justification for it in PEP257 is

> The summary line may be used by automatic indexing tools; it is important that it fits on one line and is separated from the rest of the docstring by a blank line.

Considering that sphinx has long supported 'breaking' this rule that justification no longer seems totally relevant. I've written documentation for lots of projects and really the *most* important thing is getting your point across clearly, not hard and fast rules. If a slightly longer docstring helps someone understand what a method is *without having to read the entire docstring* then it's win. I definitely agree it should be *one sentence*.